public/components/preview_panel/index.tsx

```diff
@@ -114,7 +114,7 @@ export const PreviewPanel = ({ onClose, model, dataSourceId }: Props) => {
   return (
     <EuiFlyout onClose={onCloseFlyout}>
       <EuiFlyoutHeader hasBorder>
-        <EuiTitle size="m">
+        <EuiTitle size="s">
```

Comment:
Should be `<EuiText size="s">` instead of `<EuiTitle size="s">`

Reply:
Other than this looks good!

Reply:
Sure, will update this. Thanks.